### ISSUE TITLE
Factors, numbers and coercion

### ISSUE THREAD

**Issue by rgaiacs** [swcarpentry/r-novice-gapminder#266]
Today I was playing with R and I discovery that some times transform factors and numbers strait forward can not work as you expect:

~~~
> f <- factor(c(50, 30, 10, 90, 40, 30, 50, 60, 80, 30))
> f
 [1] 50 30 10 90 40 30 50 60 80 30
Levels: 10 30 40 50 60 80 90
> as.numeric(f)
 [1] 4 2 1 7 3 2 4 5 6 2
~~~

You need to use 

~~~
> as.numeric(as.character(f))
 [1] 50 30 10 90 40 30 50 60 80 30
~~~

I think this could be a good challenge or a good call out box since in a quick look I didn't found it on the lesson.

@swcarpentry/r-maintainers please tag this one as suitable for instructor training.

**Comment by naupaka** [comment]
If it gets added, should probably be around [line 104 in Data Structures part 2](https://github.com/swcarpentry/r-novice-gapminder/blame/gh-pages/_episodes_rmd/05-data-structures-part2.Rmd#L104).

**Comment by naupaka** [comment]
@rgaiacs I've only be using the instructor-training label for PRs submitted as part of instructor training. I've added 'help-wanted' and 'newcomer friendly' to this one.

**Comment by adsteen** [comment]
I'm having trouble understanding the workflow for contributing (see issue #298) but I've emailed a .Rmd of lesson 05 that includes a challenge related to this problem. The updated .Rmd is also attached here (as a .txt since github doesn't support .Rmd). 
[05-data-structures-part2.txt](https://github.com/swcarpentry/r-novice-gapminder/files/1200904/05-data-structures-part2.txt)

**Comment by naupaka** [comment]
Hi @adsteen thanks for the comment and the change! I noticed that you are scheduled to be in the vegan workshop at ESA this Sunday that I'll be teaching - perhaps we can take a few moments to go through the contribution workflow for lessons? Might be easier than online...

**Comment by naupaka** [comment]
Resolved by #299
